apps/emqx_authz/src/emqx_authz_api_mnesia.erl

```diff
@@ -70,19 +70,19 @@ api_spec() ->
 
 paths() ->
     [
-        "/authorization/sources/built_in_database/username",
-        "/authorization/sources/built_in_database/clientid",
-        "/authorization/sources/built_in_database/username/:username",
-        "/authorization/sources/built_in_database/clientid/:clientid",
-        "/authorization/sources/built_in_database/all",
-        "/authorization/sources/built_in_database/purge-all"
+        "/authorization/sources/built_in_database/rules/users",
```

Comment:
🤔  We should not delete the existing interface directly, which will bring compatibility problems.
IMO, we should mark the existing interfaces as deprecated but still accessible. and add new interfaces under the `rules` path?

Reply:
@HJianBo the background of this task is to make as many breaking changed in our API as possible before the e5.0.0 release.

Reply:
Got it. thanks for your explanation